### PR TITLE
Use constants instead of 0, 1, 2

### PR DESCRIPTION
**Pull request by k4b7**
I find the 0, 1, 2 not very informative.  I've replace these with strings and constants in place of the strings since it's easier they're easier to read with the default VSCode syntax highlighting.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -9,25 +9,31 @@
...
+        "prefer-template": OFF,
         // We've decided explicitly not to care about this.
-        "arrow-parens": 0,
+        "arrow-parens": OFF,
         // ---------------------------------------
         // ES6/jsx stuff that's disabled for now, but maybe shouldn't be.
         // TODO(csilvers): enable these if/when community agrees on it.
-        "prefer-arrow-callback": 0,
+        "prefer-arrow-callback": OFF,
+        "prettier/prettier": [ERROR, require("./prettierrc.js")],
```
Comment (k4b7) [inline index.js]: I probably should've made this change as part of the previous PR.

File: flow.js [hunk 2]
```diff
@@ -1,33 +1,38 @@
...
+const WARN = "warn";
+const ERROR = "error";
+
 module.exports = {
     rules: {
-        "flowtype/boolean-style": [2, "boolean"],
-        "flowtype/define-flow-type": 1, // suppress no-undef on flow types
-        "flowtype/no-dupe-keys": 2,
+        "flowtype/boolean-style": [ERROR, "boolean"],
+        "flowtype/define-flow-type": "warn", // suppress no-undef on flow types
```
Comment (jeresig) [inline flow.js]: This could be `WARN`

File: flow.js [hunk 3]
```diff
@@ -1,33 +1,38 @@
...
+        "flowtype/no-dupe-keys": ERROR,
 
         // Use Flow's version of no-unused-expressions
         "flowtype/no-unused-expressions": [
             2,
```
Comment (jeresig) [inline flow.js]: This could be `ERROR`

**Review by jeresig — APPROVED** [review]
Looks good - you missed a couple so I mentioned that inline

**Comment by k4b7** [comment]
> Looks good - you missed a couple so I mentioned that inline

Thanks for catching those.

**Pull request merged**